src/bin/parser.js

```diff
@@ -244,109 +247,49 @@ export default class Harmonic {
                     pattern: ':language',
                     replacement: lang
                 }],
-                structure: getStructure(config.i18n.default, lang, config.posts_permalink)
+                structure: getStructure(config.i18n.default, lang, permalink)
             });
 
-            metadata.file = postspath + file;
+            metadata.file = filesPath + file;
             metadata.filename = filename;
-            metadata.link = postPath;
+            metadata.link = filePath;
             metadata.lang = lang;
             metadata.default_lang = config.i18n.default !== lang; // FIXME https://github.com/JSRocksHQ/harmonic/issues/169
 
-            const postHTMLFile = template
+            const contentHTMLFile = template
                 .render({
-                    post: {
+                    [filesType]: {
                         content: md.markdown(),
-                        metadata
+                        metadata: metadata
                     },
-                    config,
-                    lang
+                    config: config
                 })
                 .replace(/<!--[\s\S]*?-->/g, '');
 
+            if(filesType === 'page') {
+                metadata.content = contentHTMLFile;
+            }
+
             if (metadata.published && metadata.published === 'false') {
                 return;
             }
 
             if (metadata.date && metadata.date > currentDate) {
-                console.log(clc.info(`Skipping future post ${metadata.filename}`));
+                console.log(clc.info(`Skipping future ${filesType} ${metadata.filename}`));
                 return;
             }
 
-            const postDirPath = path.join(this.sitePath, 'public', postPath);
+            const postDirPath = path.join(this.sitePath, 'public', filePath);
             const postFilePath = path.join(postDirPath, 'index.html');
```

Comment:
I completely forgot this variables.
How about  `publicFilePath` and `publicFileDirPath`?